README.md

```diff
@@ -20,4 +20,12 @@ https://cha2251.github.io/rit-player-piano/
 # Running unit test
 - Run tests with `pytest` in top level directory
 - Follow test discovery syntax when creating new tests 
- - https://docs.pytest.org/en/7.1.x/explanation/goodpractices.html#test-discovery
\ No newline at end of file
+ - https://docs.pytest.org/en/7.1.x/explanation/goodpractices.html#test-discovery
+
+# Building Executeable
+- Start virtual environment
+- Windows:
+    - In top level directory run: `pyinstaller.exe src\common\main.py --clean --onefile`
+        - Fix permission erros with: `chmod -R -c u+rwx .`
```

Comment:
Why is this in the Windows section? Windows is inferior and doesn't have `chmod`. Also typo in 'erros'.

Reply:
I.. uh.. it works on my machine?? Now that I think about it I'm a little concerned it does tho